cypress/e2e/the-internet/the-internet-tests.cy.js

```diff
@@ -270,4 +271,17 @@ describe("The Internet Test Suite", () => {
 		floatingMenuPage.scrollToBottom();
 		floatingMenuPage.getMenu().should("be.visible");
 	});
+
+	it("Tests Hovers", () => {
+		const hoversPage = new hovers();
+		for (var index = 0; index < 3; index++) {
+			hoversPage.visit();
+			hoversPage.getProfileInfo().eq(index).should("not.be.visible");
+			hoversPage.getViewProfileButton().eq(index).should("not.be.visible");
+			hoversPage.getProfileInfo().eq(index).invoke("show");
+			hoversPage.getProfileInfo().eq(index).should("be.visible");
+			hoversPage.getViewProfileButton().eq(index).should("be.visible");
+			hoversPage.getViewProfileButton().eq(index).click();
+		}
```

Comment:
Since you are calling **_.eq(index)_** for each element fetched, maybe convert the methods to accept the index parameter in order to shorten the code in the test.